.env.template

```diff
@@ -67,7 +68,6 @@ PINECONE_ENV=your-pinecone-region
 # REDIS_PORT - Redis port (Default: 6379)
 # REDIS_PASSWORD - Redis password (Default: "")
 # WIPE_REDIS_ON_START - Wipes data / index on start (Default: False)
-# MEMORY_INDEX - Name of index created in Redis database (Default: auto-gpt)
```

Comment:
Is this deletion intentional?

Reply:
No, I must have accidentally removed it during merge. I added it back.